examples/api/src-tauri/Cargo.toml

```diff
@@ -37,14 +37,3 @@ features = [
 [dev-dependencies.tauri]
 path = "../../../core/tauri"
 features = ["test"]
-
-[features]
-prod = ["tauri/custom-protocol"]
```

Comment:
hey i liked this feature.. a super convenient way to run the app in production mode

Reply:
I didn't find it being used somewhere so just removed it, let's bring it back